### PR TITLE
Updated README to reflect file rename

### LINKED ISSUE #1093: Out of date README.md
I wanted to get more information on how to contribute to this project, and the README mentions referring to a CONTRIBUTING.md, but there is no file in this repository with that name. After a few minutes I realized this was because CONTRIBUTING.md was renamed to QUICKSTART.md in an earlier commit. 

I know this is small, but changing the README to reflect the new name could quell some potential confusion.

(I wasn't sure if I could make the change myself and submit a PR or not, but I would be happy to do so.)

### PR DESCRIPTION
**Pull request by lizzyd710**
Changed CONTRIBUTING.md to QUICKSTART.md to match the rename from an earlier commit. Also added a link to the file.

* [x] Are you working on the latest release?
* [x] Have you tested it locally?
* [x] Please explain your change
* [ ] I have opened a [Jira Issue](https://tosdr.atlassian.net/browse/EDIT) for this Pull Request
* [ ] My Squashed Commit Message Contains the Issue Key e.g `Feature: EDIT-12 Added XXX`
Thanks !

Change: I updated the README changing CONTRIBUTING.md to QUICKSTART.md since the file was renamed. I also added a link to QUICKSTART.md

(Since this was only a small fix to a README, I didn't create a Jira issue or have an issue key. Let me know if I should do this.)

Fixes #1093 #1093

### CODE REVIEW

**Comment by lizzyd710** [comment]
@Vinnl

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Phoenix is a web app to submit points to the Terms of Service; Didn't Read proje
 
 The app was developed with Ruby on Rails. The database uses PostgreSQL. You must have those if you want to run the app.
 
-Please refer to the CONTRIBUTING.md guide for more information. If anything is missing, please let us know.
+Please refer to the [QUICKSTART.md](https://github.com/tosdr/edit.tosdr.org/blob/master/QUICKSTART.md) guide for more information. If anything is missing, please let us know.
```
Comment (Vinnl) [inline README.md:9]:
```suggestion
Please refer to the [QUICKSTART.md](./QUICKSTART.md) guide for more information. If anything is missing, please let us know.
```

Comment (Vinnl) [reply README.md:9]: This will allow people to view the Markdown file locally if they're rendering the README locally as well.

**Review by Vinnl — APPROVED** [review]
Thank you!

**Pull request merged**